docs/examples/mpi/mpi_overlap_bcast_net.py

```diff
@@ -157,7 +156,7 @@
     # Report back to user
     # print(f"Sample of circuit pair {(i, j)} taken. Overlap: {overlap}")
     if rank == root and progress_bar * progress_checkpoint < k:
-        print(f"{progress_bar*10}%")
+        print(f"{progress_bar*10}%")  # noqa: T201
```

Comment:
This suppression occurs many many times; would it be better to add it as an exception in the config?